docs/en/operation/Troubleshooting.md

```diff
@@ -56,6 +56,69 @@ set the debug server's host and port, and start the debug session. If you set a
 will enter debug mode and you can inspect the current context's variables, call stack, thread list, and expression
 evaluation.
 
+## Alluxio collectInfo command
+
+Alluxio has a `collectInfo` command that collect information to troubleshoot an Alluxio cluster. 
+`collectInfo` will run a set of sub-commands that each collects one aspect of system information, as explained below.
+In the end the collected information will be bundled into one tarball which contains all information regarding your Alluxio cluster.
+The tarball can then be shared with someone you trust to help troubleshoot your Alluxio cluster.
+
+The `collectInfo` will SSH to each node and execute the set of sub-commands.
+In the end of execution the collected information will be written to files and tarballed.
+Each individual tarball will be collected to the issuing node.
+Then all the tarballs will be bundled into the final tarball, which contains all information about the Alluxio cluster. 
+
+>NOTE: Be careful if your configuration contains credentials like AWS keys!
+You should ALWAYS CHECK what is in the tarball and REMOVE the sensitive information from the tarball before sharing it with someone!
+
+### Collect Alluxio cluster information
```

Comment:
Specifically mention which of these commands has credential leaking risks. That way users can know what is and isn't safe.

Reply:
PTAL at the more detailed WARNINGs. Thanks!